source/javascripts/meetups.js

```diff
@@ -45,7 +45,9 @@
 
   handler.buildMap(mapOptions, function() {
     if(navigator.geolocation) {
-      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap);
+      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap, function(){
```

Comment:
Because the API calls the error callback with some error object as its parameter - I didn't want this to be passed in to the drawMap function